src/public/modules/forms/services/myinfo.service.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios'
```

Comment:
should we move typescript services to their own file like in `src/public/services`? Will help in (partial) migration. Should we also use camelcase style? E.g. `MyInfoService.ts`